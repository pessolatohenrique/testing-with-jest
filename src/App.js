import React from 'react';
import './App.css';
import { Provider } from "react-redux";
import store from "./store";
import Todos from './Todos';

function App() {
  return (
    <Provider store={store}>
        <div className="App">
            <Todos />
        </div>
    </Provider>
  );
}

export default App;
