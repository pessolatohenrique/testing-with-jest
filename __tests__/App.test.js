import React from 'react';
import { shallow } from 'enzyme';
import App from '../src/App';

describe('Testing app component', () => {
    it('should render correctyl', () => {
        const wrapper = shallow(
            <App />
        );

        expect(wrapper).toMatchSnapshot();
    })
});