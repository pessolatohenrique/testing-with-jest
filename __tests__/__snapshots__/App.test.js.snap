// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing app component should render correctyl 1`] = `
"<Provider store={{...}}>
  <div className=\\"App\\">
    <Connect(Todos) />
  </div>
</Provider>"
`;
